addons/osio-addon/src/main/java/io/fabric8/launcher/osio/client/AnalyticsClient.java

```diff
@@ -0,0 +1,79 @@
+package io.fabric8.launcher.osio.client;
+
+
+import java.io.IOException;
+
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+import io.fabric8.launcher.base.EnvironmentSupport;
+import io.fabric8.launcher.base.http.HttpClient;
+import io.fabric8.launcher.base.http.HttpException;
+import io.fabric8.launcher.base.identity.TokenIdentity;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import static io.fabric8.launcher.base.http.Requests.securedRequest;
+
+import static io.fabric8.utils.URLUtils.pathJoin;
+import static java.util.Objects.requireNonNull;
+
+@RequestScoped
+public class AnalyticsClient {
+
+
+    private final TokenIdentity authorization;
+
+    private final HttpClient httpClient;
+    private static final String analyticsUrl = EnvironmentSupport.getRequiredEnvVarOrSysProp("F8A_ANALYTICS_RECOMMENDER_API_URL");
```

Comment:
Don't forget to change the openshift template to get this value from the configmap (https://github.com/fabric8-launcher/launcher-backend/blob/master/openshift/template.yaml#L221-L226)

Reply:
The attribute name needs to be in uppercase

Reply:
ok

Reply:
Also make sure that the configmap entry is created in production and prod-preview before we merge these changes

Reply:
How to do that?

Reply:
Also is there a reason why this attribute is not in `OsioConfigs`?

Reply:
No, I don't know the reason why it's not there.

Reply:
Can you move it over there? It's nice to have a single place where all the environment vars are listed